packages/node_modules/pouchdb-core/src/setup.js

```diff
@@ -127,4 +128,6 @@ PouchDB.fetch = function (url, opts) {
   return fetch(url, opts);
 };
 
+PouchDB.prototype.activeTasks = PouchDB.activeTasks = new ActiveTasks();
```

Comment:
What's the effect of assigning to `PouchDB.activeTasks` if you have multiple `PouchDB` instances around, or if a new instance is implicitly created to store a view index?

Reply:
I agree that this approach is a bit confusing; the original sketch had the active tasks API reside in `PouchDB.activeTasks`, but that way we can't access the object in `AbstractPouchDB` and other elements that compose the `PouchDB` object. So for technical reasons, we use `PouchDB.prototype.activeTasks` to make it accessible to other parts of the inheritance hierarchy of the PouchDB class.

On a semantic level, I find it sensible to monitor all PouchDB activity of a particular application rather than monitoring only certain instances.

Technically, we could drop the `PouchDB.activeTasks` reference and just keep the prototype reference in order to keep it simple.